cache_checker.js

```diff
@@ -122,16 +122,21 @@ is_cached = async (URL, flag) => {
 
   //Print the Results
   console.log(dataHashMap);
+  console.log(
+    "Total Request Made:",
+    dataHashMap.count.cssCount +
+      dataHashMap.count.imageCount +
+      dataHashMap.count.fontCount +
+      dataHashMap.count.jsCount
+  );
   console.log(is_leverage_cache);
   browser.close();
-  console.log('Completed!');
+  console.log("Completed!");
 
   /* To check if the URL has SSL certificate */
-  const options = updatedURL;
-
+  const options = URL;
   var client = http; // To check if the http has a SSL certificate
-
-  if (updatedURL.indexOf('https') === 0) {
+  if (URL.indexOf("https") === 0) {
```

Comment:
here as well

Reply:
@sanchitgupta001 I have changed code as you said.Can you please check again?